src/app/tests/pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+asyncio_mode = strict
```

Comment:
Please add an explanatory comment on why we need this mode

Reply:
I added a comment, at least in what it does. Also made it more lax with `auto`. I don't remove the `pytest.asyncio` in the files as I feel that it is cleaner. It allows us theoretically to use other async test frameworks, when the mode is strict. But I am quite sure we won't need another. The annotations are practically a leftover, but see no reason to remove them.